Extensions/3D/DirectionalLight.ts

```diff
@@ -74,6 +74,16 @@ namespace gdjs {
               this.updateRotation();
             }
           }
+          getDoubleParameter(parameterName: string): number {
+            if (parameterName === 'intensity') {
+              return this.light.intensity;
+            } else if (parameterName === 'elevation') {
+              return this.elevation;
+            } else if (parameterName === 'rotation') {
+              return this.rotation;
```

Comment:
No conversion needed for those 2 angles?

Reply:
Yes, I've been to quick to remove it. There were clamping at other places. I removed it by habit.